ansible/roles/druid-ingestion/templates/telemetry_audit_rollup_syncts_kafka

```diff
@@ -0,0 +1,680 @@
+            },
+            {
+              "type": "string",
+              "name": "collection_created_by"
+            },
+            {
+              "name": "collection_created_for"
+            },
+            {
+              "type": "string",
+              "name": "user_type"
+            },
+            {
+              "type": "string",
+              "name": "user_signin_type"
+            },
+            {
+              "type": "string",
+              "name": "user_login_type"
+            },
+            {
+              "type": "long",
+              "name": "device_first_access"
+            },
+            {
+              "type": "string",
+              "name": "user_declared_state"
+            },
+            {
+              "type": "string",
+              "name": "device_loc_district"
+            },
+            {
+              "type": "string",
+              "name": "derived_loc_from"
+            },
+            {
+              "name": "l2data_created_for"
+            },
+            {
+              "type": "string",
+              "name": "l2data_framework"
+            },
+            {
+              "type": "string",
+              "name": "l2data_name"
+            },
+            {
+              "type": "string",
+              "name": "l2data_channel"
+            },
+            {
+              "type": "string",
+              "name": "l2data_mimetype"
+            },
+            {
+              "name": "l2data_medium"
+            },
+            {
+              "type": "string",
+              "name": "l2data_board"
+            },
+            {
+              "type": "string",
+              "name": "l2data_contenttype"
+            },
+            {
+              "name": "l2data_gradelevel"
+            },
+            {
+              "name": "l2data_subjects"
+            }
+          ],
+          "dimensionsExclusions": []
+        },
+        "timestampSpec": {
+          "column": "syncts",
+          "format": "auto"
+        }
+      }
+    },
+    "transformSpec": {
+      "transforms": [
+        {
+          "type": "expression",
+          "name": "device_first_access",
+          "expression": "timestamp_format(device_first_access, 'YYYY-MM-dd','Asia/Kolkata')"
+        }
+      ]
+    },
+    "metricsSpec": [
+      {
+        "name": "total_count",
+        "type": "count"
+      },
+      {
+        "type": "doubleSum",
+        "name": "total_edata_duration",
+        "fieldName": "edata_duration"
+      }
+    ],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "DAY",
+      "rollup": true,
+      "intervals": null
+    }
+  },
+  "ioConfig": {
+    "topic": "{{env}}.telemetry.audit",
+    "consumerProperties": {
+      "bootstrap.servers": "{{kafka_brokers}}"
+    },
+    "taskCount": 1,
+    "replicas": 1,
+    "taskDuration": "PT14400S",
+    "useEarliestOffset": false
```

Comment:
it should be similar to telemetry_feedback_events spec. It should filter for audit events as below
"transformSpec": {
      "filter": {
        "type": "selector",
        "dimension": "eid",
        "value": "AUDIT"
      }
    }